utilities/zookeeper/R/corrector-aardvark.R

```diff
@@ -44,6 +44,52 @@ default_state_params <- function(
   )
 }
 
+# 
+special_flags_marker <- function(
+  geo_value = NA,
+  signal = NA,
+  time_value = NA,
+  max_lag = NA) {
+  tibble::tibble(
+    geo_value = geo_value,
+    signal = signal,
+    time_value = time_value,
+    max_lag = max_lag
+  )
+}
+
+#' @examples
+#' special_flags <- special_flags_marker(c('wa','oh'), 
+#'                                      c('deaths_incidence_num','deaths_incidence_num'),
+#'                                      list(lubridate::ymd(c("2020-12-16", "2020-12-17","2020-12-23", "2020-12-24","2020-12-29")),
+#'                                            seq(lubridate::ymd("2021-02-20"), lubridate::ymd(time_value_flag_date),by=1)),
+#'                                      c(14,60))
+#'    
+#'             
+#'  Two helper functions
+#'  (1) Add special flagging columns, this function will create columns like "flag_bad_va", "flag_bad_wa", etc.    
+special_flagging <- function(df,special_flags){
```

Comment:
Did you run the local R CMD Check? Does it generate documentation for these? This one needs to be exported  and fully documented since we're going to have to call it. I would suggest (1) copy the current documentation and then delete; (2) putting the cursor in the function body and press Cmd+Opt+Shift+R to get a documentation skeleton.

Reply:
Actually, which ones of these do we need to export? I think it's only `special_flags_marker()` right? Let's rename it as `make_manual_flags()` so that its name matches the other functions.

Reply:
> Did you run the local R CMD Check? Does it generate documentation for these? This one needs to be exported and fully documented since we're going to have to call it. I would suggest (1) copy the current documentation and then delete; (2) putting the cursor in the function body and press Cmd+Opt+Shift+R to get a documentation skeleton.

We do not need to export these two helper functions.